relay/kusama/src/lib.rs

```diff
@@ -1735,6 +1737,18 @@ pub mod migrations {
 		}
 	}
 
+	/// Enable the elastic scaling node side feature.
+	///
+	/// This is required for Coretime to ensure the relay chain processes parachains that are
+	/// assigned to multiple cores.
+	pub struct EnableElasticScalingNodeFeature;
+	impl OnRuntimeUpgrade for EnableElasticScalingNodeFeature {
+		fn on_runtime_upgrade() -> Weight {
+			let _ = Configuration::set_node_feature(RawOrigin::Root.into(), 1, true);
+			Weight::from_parts(1, 0)
```

Comment:
why do we not returning the actual weight of the extrinsic?

Reply:
Because the function doesn't return it, but I got some idea